lib/elixir/lib/inspect/algebra.ex

```diff
@@ -565,6 +600,9 @@ defmodule Inspect.Algebra do
     end
   end
 
+  defp ansi(color),
+    do: IO.ANSI.format_fragment([color], true)
```

Comment:
This should be `color` and we should make sure to call `List.wrap()` when passing the color to `doc_color` so we allow people to set up multiple color settings. :)